docs/en_US/install/3ds.md

```diff
@@ -102,3 +109,30 @@ You may have thought to yourself; _"2nd local account? What's that? I thought th
 <ul id="footnotes">
 	<li id="footnote-1"><sup>[1]</sup> Some games may require a PNID for certain actions, such as eShop purchases. The only known game which requires a PNID for general use is Nintendo Badge Arcade, which is not yet supported</li>
 </ul>
+
+### What do I do if my console says that my badges get erased?
```

Comment:
This is sort of an odd header. If you deleted your badges by opening badge arcade after switching accounts, this process isn't likely to help you recover them. I think badge arcade sees your NNID and says "that's not who this console is registered to anymore" and just re-initalizes an empty badge database. 

A better header would be something like "How do I transfer my badges to my PNID?" or "What do I do if the Home Menu says I have no badges after switching?"